mkdocs.yml

```
@@ -89,7 +89,7 @@ nav:
   - 'Exporting Models': usage/exporting_models.md
   - 'Computed Fields': usage/computed_fields.md
   - 'Dataclasses': usage/dataclasses.md
-  - 'Validation Decorator': usage/validation_decorator.md
+  - '@validate_call Decorator': usage/validation_decorator.md
```

I think the "Validation" from before is more cute. 🤔 
```suggestion
  - 'Validation Decorator': usage/validation_decorator.md
```